dbfit-java/core/src/main/java/dbfit/fixture/StatementExecution.java

```diff
@@ -4,27 +4,20 @@
 
 public class StatementExecution implements AutoCloseable {
     protected PreparedStatement statement;
-    protected int returnValueInd = -1;
 
-    public StatementExecution(PreparedStatement statement, boolean clearParameters) {
+    public StatementExecution(PreparedStatement statement) {
```

Comment:
> Should/could we allow this to take the SQL text directly instead of a PreparedStatement?

Well, `StaementExecution` doesn't know how to create statements from SQL; can be solved by passing some extra factory parameter but that won't make things simpler.